dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -281,6 +283,12 @@
 				status = "disabled";
 				#pwm-cells = <3>;
 			};
+
+			qdec {
+				compatible = "st,stm32-qdec";
+				status = "disabled";
+				input-filter-level = <NO_FILTER>;
```

Comment:
should property be prefixed with `st,`? @erwango

Reply:
Exact, thanks for noticing.